tests/system/cases/datasources/main.tf

```diff
@@ -0,0 +1,61 @@
+provider "layer0" {
+  endpoint        = "${var.endpoint}"
+  token           = "${var.token}"
+  skip_ssl_verify = true
+}
+
+resource "layer0_environment" "datasources" {
+  name = "dsrctest"
+}
+
+resource "layer0_load_balancer" "datasources" {
+  name        = "dsrctest_lb"
```

Comment:
[nit] personal preference, but I don't like putting entity-type suffix in names since it's redundant. It makes since for introductory walkthroughs, but not a pattern I'd expect/want seasoned layer0 users to do.